src/pages/index.js

```diff
@@ -83,10 +219,62 @@ addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
 
-// Form validation setup
-document
-  .querySelectorAll(validationConfig.formSelector)
-  .forEach((formElement) => {
-    const formValidator = new FormValidator(validationConfig, formElement);
-    formValidator.enableValidation();
-  });
+document.querySelectorAll(validationConfig.formSelector).forEach((formElement) => {
+  const formValidator = new FormValidator(validationConfig, formElement);
+  formValidator.enableValidation();
```

Comment:
Please, delete it because you have created separate validators in the code, otherwise you add listeners again and again to the same elements. This can cause a **`memory leak`**